shared/lib/transactions-controller-utils.js

```diff
@@ -35,6 +35,52 @@ export function calcTokenAmount(value, decimals) {
   return new BigNumber(String(value)).div(multiplier);
 }
 
+export function stripAddressFromLogTopic(addressLogTopic) {
+  return `0x${addressLogTopic.replace(/^0x000000000000000000000000/u, '')}`;
+}
+
+/**
+ * Extracts token transfers from a txReceipt by filtering through logs.
+ *
+ * @param txReceipt - A transaction receipt.
+ * @returns {{ contractAddress: string, to: string, from: string }} A list of token transfers from a txReceipt.
+ */
+export function getTokenTransfersFromTxReceipt(txReceipt) {
```

Comment:
I want to make sure this only returns ERC-20 tokens and not ERC-721 NFTs.  There are some existing issues where NFT contracts mistakenly end up in the token state instead of the NFT state.

Since both ERC-20 and ERC-721 transfer events hash to `0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef` I wonder if this also needs as check for `topics.length === 3` to distinguish them.